tests/Formatter/SmartJsonFormatterTest.php

```diff
@@ -95,4 +98,129 @@ public function test_format_ignore_paths(): void
             $this->assertStringNotContainsString($excludedPath, $trace['file']);
         }
     }
+
+    /**
+     * @covers ::addStackTraceModifier
+     * @covers ::format
+     * @covers ::RedactArgumentsMap
+     */
+    public function test_format_map_redact_arguments(): void
+    {
+        $redactedFrameArguments = [
```

Comment:
So are we telling the class what strings to never show, rather than telling it which class/function to not display args for?

Reply:
@maxakropolis Yeah. This seemed like a simpler approach.

Reply:
It makes sense as a think to be able to do but I think it would be nice if we had the other filtering option too. This is a nice improvement though - how about we go with this and add an issue to add another filter later?

Reply:
@maxakropolis We can pass a callable to `addMap()` and `addFilter()`, so anything we need to filter beyond what's here we can do. We don't even have to include the two Map/Filter classes I've written here, we could just move them into our own code and adjust them as needed.

Reply:
I think it's nice to include them as an example